include/crocoddyl/core/activations/quadratic-log.hpp

```diff
@@ -0,0 +1,75 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2018-2019, LAAS-CNRS
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef CROCODDYL_CORE_ACTIVATIONS_QUADRATIC_LOG_HPP_
+#define CROCODDYL_CORE_ACTIVATIONS_QUADRATIC_LOG_HPP_
+
+#include <stdexcept>
+#include "crocoddyl/core/fwd.hpp"
+#include "crocoddyl/core/activation-base.hpp"
+#include "crocoddyl/core/utils/exception.hpp"
+#include <iostream>
+
+namespace crocoddyl {
+
+template <typename _Scalar>
+class ActivationModelQuadLogTpl : public ActivationModelAbstractTpl<_Scalar> {
+ public:
+  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
+
+  typedef _Scalar Scalar;
+  typedef MathBaseTpl<Scalar> MathBase;
+  typedef ActivationModelAbstractTpl<Scalar> Base;
+  typedef ActivationDataAbstractTpl<Scalar> ActivationDataAbstract;
+  typedef typename MathBase::VectorXs VectorXs;
+  typedef typename MathBase::MatrixXs MatrixXs;
+
+  explicit ActivationModelQuadLogTpl(const std::size_t& nr,const Scalar& sigma2)
+     : Base(nr), sigma2_(sigma2){};
+  virtual ~ActivationModelQuadLogTpl(){};
+
+  virtual void calc(const boost::shared_ptr<ActivationDataAbstract>& data, const Eigen::Ref<const VectorXs>& r) {
```

Comment:
Did you format the code?
This code looks that need to be formatted.
See the wiki for more information about how to do it.

Reply:
I tried to clone Gepetto Linter in my home to format the code but apparently I don't have access to the remote repository

Reply:
this repository should be public: https://github.com/gepetto/linters
the docker image too : `docker pull gepetto/linters`